content/03-reference/01-tools-and-interfaces/02-prisma-client/050-filtering.mdx

```diff
@@ -522,6 +524,35 @@ const result = await prisma.user.findMany({
 })
 ```
 
+## Case-insensitive filtering
+
+Case-insensitive filtering is preview feature and [can only be used with PostgreSQL](case-sensitivity#options-for-case-insensitive-filtering). Add the highlighted feature flag to enable distinct querying capabilities:
+
+generator client {
+provider = "prisma-client-js"
+previewFeatures = ["insensitiveFilters"]
+}
```

Comment:
````suggestion
```prisma
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["insensitiveFilters"]
}
```
````

Reply:
Not sure how to highlight a line.